src/plugins/navigation/review/review.js

```diff
@@ -323,7 +324,7 @@ export default pluginFactory({
         });
 
         this.explicitlyHidden = false;
-        this.customLayout = isCustomLayout();
+        this.isFizzyLayout = isFizzyLayout();
```

Comment:
I'm curious why you call the function and assign its result so late in `init`? Couldn't you call it higher up and then use `this.isFizzyLayout` everywhere else, instead of the function call?

Anyway, it's not a big deal but looks like not the simplest way.